packages/e2e-tests/examples.json

```diff
@@ -42,5 +42,6 @@
   "doc_minified_chromium.html": "5cdb2ffe-beff-4d35-9689-9242bd277f2d",
   "cypress-realworld/bankaccounts.spec.js": "6d4abbe1-2fcb-47d9-92d5-f7b45760b01a",
   "flake/adding-spec.ts": "7b0db00a-d9a9-4d2d-b816-af14a1b682a6",
-  "breakpoints-01": "f5fa8af9-70a4-4c9c-811f-1388b62c0b15"
-}
\ No newline at end of file
+  "breakpoints-01": "f5fa8af9-70a4-4c9c-811f-1388b62c0b15",
```

Comment:
Hey @markerikson @Mokshit06 How does the underlying recording for this test get regenerated? It looks different from the others in the `examples.json` file.